pkg/controller/volume/attachdetach/cache/desired_state_of_world.go

```diff
@@ -46,7 +46,7 @@ type DesiredStateOfWorld interface {
 	// AddNode adds the given node to the list of nodes managed by the attach/
 	// detach controller.
 	// If the node already exists this is a no-op.
-	AddNode(nodeName k8stypes.NodeName)
+	AddNode(nodeName k8stypes.NodeName, keepTerminatedPodVolumes bool)
```

Comment:
nit: Update comment to describe the new parameter. This is particularly important for interfaces so that callers and implementers know what the expectations are.